src/components/application_manager/src/commands/hmi/get_urls.cc

```diff
@@ -37,57 +37,59 @@
 namespace application_manager {
 namespace commands {
 
-GetUrls::GetUrls(const MessageSharedPtr& message)
-  : RequestFromHMI(message) {
-}
+GetUrls::GetUrls(const MessageSharedPtr& message) : RequestFromHMI(message) {}
 
-GetUrls::~GetUrls() {
-}
+GetUrls::~GetUrls() {}
 
 void GetUrls::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
-  smart_objects::SmartObject& object = *message_;
-  object[strings::params][strings::message_type] = MessageType::kResponse;
-  if (application_manager::ApplicationManagerImpl::instance()->GetPolicyHandler().PolicyEnabled()) {
-    policy::EndpointUrls endpoints;
-    application_manager::ApplicationManagerImpl::instance()->GetPolicyHandler().GetServiceUrls(
-        object[strings::msg_params][hmi_request::service].asString(), endpoints);
-    if (!endpoints.empty()) {
-      object[strings::msg_params].erase(hmi_request::service);
+  using namespace smart_objects;
+  using namespace application_manager;
+  using namespace strings;
+  using namespace hmi_apis;
```

Comment:
@dev-gh  You may not use a using-directive to make all names from a namespace available. 
`// Forbidden -- This pollutes the namespace.
using namespace foo;` [Google Coding Style/Namespaces](https://google.github.io/styleguide/cppguide.html#Namespaces)